spec/bundler/bundler_spec.rb

```diff
@@ -3,6 +3,41 @@
 require 'bundler'
 
 describe Bundler do
+  describe "version 1.99" do
+    context "when bundle is run" do
+      it "should print a single deprecation warning" do
+        # install_gemfile calls `bundle :install, opts`
+        install_gemfile <<-G
+          source "file://#{gem_repo1}"
+          gem "rack"
+        G
+
+        expect(err).to lack_errors
+      end
+    end
+
+    context "when Bundler.setup is run in a ruby script" do
+
+      it "should print a single deprecation warning" do
+        install_gemfile <<-G
+          source "file://#{gem_repo1}"
+          gem "rack", :group => :test
+        G
+
+        ruby <<-RUBY
+          require 'rubygems'
+          require 'bundler'
+          Bundler.setup
```

Comment:
To verify the behavior we're looking for, we should have two lines of `Bundler.setup` here.